sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md

```diff
@@ -12,6 +14,8 @@
 
 ## 6.0.0-beta.2 (2022-09-30)
 
+Please refer to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-java/blob/4f0af481662eebd1703a4082eea9a291b76c1608/sdk/spring/CHANGELOG.md#600-beta2-2022-09-30) for more details.
```

Comment:
1. We should always update the commit id (4f0af481662eebd1703a4082eea9a291b76c1608) to the latest one. Then the link will display all release versions. 
2. We should add prefix of CHANGELOG to spring/CHANGELOG to make it less confuses without open the link

Of course, it's not caused by this PR. Could you please create an issue? And add a task item in that issue: Update release wiki.

Reply:
Opened https://github.com/Azure/azure-sdk-for-java/issues/31930